docs/topics/snippets.md

```diff
@@ -298,10 +298,11 @@ class Advert(RevisionMixin, models.Model):
 
     @property
     def revisions(self):
+        # Some custom logic here if necessary
```

Comment:
This isn't required, and the `GenericRelation` can be defined as the `revisions` override directly. The `RevisionMixin.revisions` property exists to allow using `.revisions` generically without having to define the `GenericRelation`, and also to allow any custom logic (e.g. use the base content type for Pages).

I added this detail to prevent developers mistakingly thinking this is required.